### ISSUE TITLE
Release 2021.08.0

### ISSUE THREAD

**Issue by jrbourbeau** [dask/community#176]
If there aren't any blockers, I'd like to release `dask` and `distributed` 2021.08.0 this Friday.

https://github.com/dask/dask/pull/7969 is close to done, so I'd like to try and get that in. Are there other PRs that are almost done that we'd like to get into this release? 

cc @jsignell @jakirkham @quasiben

**Comment by quasiben** [comment]
No blockers as far as I know

**Comment by jacobtomlinson** [comment]
In dask/distributed#5188 I've added a bunch of HTML repr widget tooling. I've since raised dask/dask#8019 and dask/distributed#5205 to upstream most of this to `dask` instead.

It would be great to get these merged before release so that the first release that contains this already has everything upstreamed.

**Comment by jsignell** [comment]
There are some IO issues that it would be good to sort out, but they didn't come in between the last release and this, so probably not essential: https://github.com/dask/dask/issues/8022, https://github.com/dask/dask/issues/8023

**Comment by jsignell** [comment]
I have a trivial PR for the hdf one, so I'm hoping to merge that once it passes https://github.com/dask/dask/pull/8033

**Comment by fjetter** [comment]
I just opened another PR which fixes another deadlock https://github.com/dask/distributed/pull/5210
No review, yet but if somebody could take a look and everything is fine this would be a good addition to the release.

**Comment by rjzamora** [comment]
I submitted a pretty-simple fix for #8022 in #8035

**Comment by jrbourbeau** [comment]
Alright, https://github.com/dask/dask/pull/8033, https://github.com/dask/distributed/pull/5188, and https://github.com/dask/dask/pull/7969 are in and https://github.com/dask/dask/pull/8035 is almost there (waiting on CI). There are still outstanding review comments for https://github.com/dask/distributed/pull/5210, https://github.com/dask/dask/pull/8019, and https://github.com/dask/distributed/pull/5205, so I'm inclined to hold off on those and include them in the next release.

**Comment by mrocklin** [comment]
I'll suggest that we should get used to not getting things in last minute,
and become more comfortable with releasing earlier in the day and then
waiting a couple of weeks (or one week, if anyone wants to push things
early).

**Comment by jrbourbeau** [comment]
That's a fair point -- I do feel a little uneasy merging a bunch of stuff right before releasing. Though I suspect most folks are understanding when things don't make it in close to a release. In the particular case of https://github.com/dask/dask/pull/8035, I'm willing to wait on it since it fixes an issue identified in our yet-to-be-released ORC refactor (xref https://github.com/dask/dask/pull/7756) and I'd prefer to ship that out with the patch included.

**Comment by jrbourbeau** [comment]
Closing as this was pushed out last Friday. Thanks all!
